src/main/java/mpicbg/spim/postprocessing/deconvolution2/LRFFT.java

```diff
@@ -441,7 +441,7 @@ public void setKernel( final Image<FloatType> kernel )
 				final FourierConvolution<FloatType, FloatType> fftConv = fftConvolution1;
 				fftConv.replaceImage( image );
 				fftConv.process();
-				System.out.println( " block " + i + ": compute " + (System.currentTimeMillis() - time) );
+				System.out.println(" block " + iteration + ": compute " + (System.currentTimeMillis() - time) );
```

Comment:
I fear that @StephanPreibisch's coding style disagrees with the rest of Fiji's, instead opting for some style concocted in the ImgLib2 project that is a variation of *some* C++ projects' coding style (and unfortunately, it is not used consistently, as can be seen in the hunk above where function bodies consisting of one line are on the same line as the function declaration, which violates that coding style).

The proper solution would be to document the coding style of *this* Fiji plugin (as it is an outlier when it comes to Fiji's source code), or to switch to [Fiji's source code conventions](http://fiji.sc/Coding_style).

Until that happens, I fear that the closest indication as to @StephanPreibisch's wishes is the surrounding code.